frontend/app/src/scenes/Home/scenes/UpsertCategory/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import {
+  Button,
+  Col,
+  Form,
+  Input,
+  notification,
+  Row,
+  Spin,
+  Typography,
+} from 'antd';
+import mainStyles from '../../styles.css';
+import { AddCategoryStateProps, CategoryUrlParams } from './interfaces';
+import { RouteComponentProps, withRouter } from 'react-router';
+import axios, { AxiosResponse, Method } from 'axios';
+import { Category } from '../../interfaces';
+import { handleApiError } from '../../../../services/util/errorHandler';
+
+const { Title, Text } = Typography;
+
+class AddCategory extends React.Component<
```

Comment:
Rename the class also

Reply:
Sure!

Reply:
Resolved with commit [7d2ab6e](https://github.com/sef-global/sef-core/pull/81/commits/7d2ab6e714f90975ff227736c6d722760c7c27be)